README.rst

```diff
@@ -40,6 +40,17 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb with an admin user:
```

Comment:
IIUC what is called cluster is when you deploy several InfluxDB instances with a relay in front of them. So all instances are independent but they are synchronized. But I agree that we can clarify the README.